.gitignore

```diff
@@ -17,4 +17,5 @@
 
 # build directories
 bin/
-dist/
\ No newline at end of file
+dist/
+cert/
```

Comment:
was this needed? you ended up using testdata instead

Reply:
yes during local testing for cert, i don't want this folder to be seen in git status.